packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -76,6 +84,20 @@ export function trackInteractionToNextPaint(
       } else {
         interactionToNextPaintTargetSelector = undefined
       }
+      if (
+        !interactionToNextPaintTargetSelector &&
+        isExperimentalFeatureEnabled(ExperimentalFeature.NULL_INP_TELEMETRY)
+      ) {
+        if (interactionSelectorMap.has(newInteraction.startTime)) {
+          interactionToNextPaintTargetSelector = interactionSelectorMap.get(newInteraction.startTime)
+          interactionSelectorMap.delete(newInteraction.startTime)
+        }
+      }
+      interactionSelectorMap.forEach((_, key) => {
+        if (key > 10 * ONE_SECOND) {
+          interactionSelectorMap.delete(key)
+        }
+      })
```

Comment:
**💬 suggestion:** ‏Not all browsers support INP, so if we are clearing the map when an interaction is notified, it might leak memory on browsers not supporting it.

Move this in `trackClickAction`? It could be:

```ts
function addSelectorToInteractionMap(timeStamp, selector) {
  interactionSelectorMap.set(timeStamp, selector)
  // then clear the map
}

```

Reply:
I don't see this change, normal?